.github/workflows/docker-indexer-grpc-test.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       VALIDATOR_IMAGE_REPO: ${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}/validator
       FAUCET_IMAGE_REPO: ${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}/faucet
       INDEXER_GRPC_IMAGE_REPO: ${{ secrets.GCP_DOCKER_ARTIFACT_REPO }}/indexer-grpc
-      IMAGE_TAG: ${{ inputs.GIT_SHA || 'f4100b21da4e9ba10fadd184e92e3d1c22bc282e' }} # hardcode to a known good build when not running on workflow_call
+      IMAGE_TAG: ${{ inputs.GIT_SHA || '3e29accd9d9aa2ca4802bcfa35847d13c3af7ccd' }} # hardcode to a known good build when not running on workflow_call
```

Comment:
Lets make this a GH secret at least so that we can change without a code change

Reply:
Actually I'll just change it to `devnet` tag, which should be up to date enough